examples/experimental/prompts_dev/synthetic_openai_data.ipynb

```diff
@@ -37,6 +45,18 @@
     "from weave.syndata_mon import random_predictions"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61b9c12a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "WB_ENTITY = \"\" # replace with your W&B username or team name\n",
```

Comment:
minor/nit: I set this to WB_ENTITY = so the cell errors, WB_ENITY = "" will let the cell execute & have weird behavior

Reply:
done